.circleci/config.yml

```diff
@@ -0,0 +1,56 @@
+# PHP CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-php/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # Specify the version you desire here
+      - image: circleci/php:7.3-node-browsers
```

Comment:
Any chance we can bump to 7.4 ?

Reply:
Will bump to 7.4 but we will need to confirm its use with the stakeholders